src/shared/telemetries/telemetriesconfig/config.go

```diff
@@ -9,8 +9,8 @@ const (
 	TelemetryAPIAddressKey         = "telemetry-address"
 	TimeoutKey                     = "telemetry-client-timeout"
 	CloudClientTimeoutDefault      = "30s"
-	TelemetryEnabledKey            = "telemetry-enabled"
-	TelemetryEnabledDefault        = false
+	TelemetryUsageEnabledKey       = "telemetry-usage-enabled"
```

Comment:
I'd keep the old flag as is or add a deprecated alias for it to avoid breaking existing deployments, but please double-check with ori (since it's a product definition question)

Reply:
In the helm I will anyway pass `telemetry-errors-enabled` so it will be broken if the image isn't updated. This will be a totally separated chart version since it's a breaking change, so I'm not worry

Reply:
We will keep the old flag as global enabler for the whole